api/orchestrator/orchestrator.go

```diff
@@ -28,21 +28,57 @@ package orchestrator
 import (
 	"context"
 	"errors"
+	"fmt"
+
+	"clouditor.io/clouditor/api/assessment"
 )
 
 type CloudServiceHookFunc func(ctx context.Context, cld *CloudService, err error)
 
 var (
-	ErrCertificateIsNil   = errors.New("certificate is empty")
-	ErrServiceIsNil       = errors.New("service is empty")
-	ErrNameIsMissing      = errors.New("service name is empty")
-	ErrIDIsMissing        = errors.New("service ID is empty")
-	ErrCertIDIsMissing    = errors.New("certificate ID is empty")
-	ErrCatalogIsNil       = errors.New("catalog is empty")
-	ErrCatalogIDIsMissing = errors.New("catalog ID is empty")
-	ErrToEIDIsMissing     = errors.New("toe ID is empty")
+	ErrCertificateIsNil      = errors.New("certificate is empty")
+	ErrServiceIsNil          = errors.New("service is empty")
+	ErrNameIsMissing         = errors.New("service name is empty")
+	ErrIDIsMissing           = errors.New("service ID is empty")
+	ErrCertIDIsMissing       = errors.New("certificate ID is empty")
+	ErrCatalogIsNil          = errors.New("catalog is empty")
+	ErrCatalogIDIsMissing    = errors.New("catalog ID is empty")
+	ErrToEIDIsMissing        = errors.New("toe ID is empty")
+	ErrRequestCloudServiceID = errors.New("error in cloud service id")
 )
 
+// Validate validates the UpdateMetricConfigurationRequest
+func (req *UpdateMetricConfigurationRequest) Validate() error {
+	// Check cloud service ID
+	err := assessment.CheckCloudServiceID(req.CloudServiceId)
+	if err != nil {
+		return fmt.Errorf("%s: %w", ErrRequestCloudServiceID, err)
```

Comment:
I would also just return the `err` here without any wrapping, I think this is necessary. and then we can get rid of ErrRequestCloudServiceID